modules/geolocation/geolocationWorker.py

```diff
@@ -0,0 +1,89 @@
+from modules.geolocation.geolocation import Geolocation
+import multiprocessing as mp
+# For exception handling
+import queue
+
+
+def exit_requested(requestQueue):
+    return not requestQueue.empty()
+
+
+def geolocation_worker(pause, exitRequest, pipelineIn, pipelineOut, pipelineOutLock):
+
+    print("Start Geolocation")
+    locator = Geolocation()
+
+    while True:
+
+        pause.acquire()
+        pause.release()
+
+        # Pixel coordinates of tents and plane data
+        coordinates = pipelineIn.get()
+        # TODO verify this
+        if (not coordinates):
```

Comment:
This one's an interesting case, for now assume an empty list or some other falsey value, I'll have to see how our targetAcquisition module handles exceptions and invalid output.